pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/IngestionDelayTracker.java

```diff
@@ -174,6 +188,22 @@ private long getIngestionDelayMs(long ingestionTimeMs) {
     return agedIngestionDelayMs;
   }
 
+  private long getPartitionOffsetLag(IngestionOffsets offset) {
+    if (offset == null) {
```

Comment:
Curious, can this even be null?

Reply:
yeah we pass that when we want to reset the metric